controllers/status.go

```diff
@@ -60,7 +61,7 @@ func setVRGInitialCondition(conditions *[]metav1.Condition, observedGeneration i
 	})
 }
 
-// sets conditions when VRG is data replicating
+// sets conditions when Secondary VRG is data replicating
```

Comment:
I don't understand this comment for a Secondary VRG.

Reply:
For Primary we are changing the reason from "Replicating" to "Ready". So the reason of "Replicating" is specific to Secondary and the below function is called when VRG is secondary. Probably I will add bit more context in the comment to make it more clear.